python/thunder/imgprocessing/transformation.py

```diff
@@ -0,0 +1,58 @@
+""" Transformations produced by registration methods """
+
+
+class Transformation(object):
+    """ Base class for transformations """
+
+    def apply(self, im):
+        raise NotImplementedError
+
+
+class Displacement(Transformation):
+
+    def __init__(self, delta=None):
+        self.delta = delta
+
+    def apply(self, im):
+        """
+        Apply an n-dimensional displacement by shifting an image or volume.
+
+        Parameters
+        ----------
+        im : ndarray
+            The image or volume to shift
+        """
+        from scipy.ndimage.interpolation import shift
+
+        return shift(im, map(lambda x: -x, self.delta), mode='nearest')
```

Comment:
How would you feel about storing the deltas as numpy arrays instead of tuples? This could clean up a lot of these map calls.